openedx/core/djangoapps/zendesk_proxy/utils.py

```diff
@@ -0,0 +1,67 @@
+"""
+Utility functions for zendesk interaction.
+"""
+import json
+import logging
+from urlparse import urljoin
+
+from django.conf import settings
+import requests
+
+log = logging.getLogger(__name__)
+
+
+def create_zendesk_ticket(requester_name, requester_email, subject, body, tags=None):
+    """
+    Create a Zendesk ticket via API.
+
+    Note that we do this differently in other locations (lms/djangoapps/commerce/signals.py and
+    common/djangoapps/util/views.py). Both of those callers use basic auth, and should be switched over to this oauth
+    implementation once the immediate pressures of zendesk_proxy are resolved.
+    """
+    if not (settings.ZENDESK_URL and settings.ZENDESK_OAUTH_ACCESS_TOKEN):
+        log.debug('Zendesk is not configured. Cannot create a ticket.')
+        return
+
+    # Remove duplicates from tags list
+    tags = list(set(tags))
```

Comment:
@fysheets pls note that I've now modified the expected input so that `tags` contains an array of values.